src/components/Calculator.js

```diff
@@ -1,55 +1,163 @@
 /** @format */
```

Comment:
- [ ] _I can't find the readme file in commits so I will add my comment here but it is intended for the README file_:
- The picture in the readme file is broken kindly fix it
- If you do not intend to add a live link of your project kindly remove that part from readme